### PR TITLE
Add doc about perf comparison

### PR DESCRIPTION
**Pull request by snuyanzin**
The PR is aiming to show comparison of performance between datafaker and javafaker,  jFairy, kotlin-Faker across different jdks.
There is a separate project https://github.com/snuyanzin/datafaker-benchmark which could be used for some measurements

### CODE REVIEW

**Comment by bodiam** [comment]
@snuyanzin Good idea. What also might be interesting is to compare it other libraries, like kotlin-faker:

https://github.com/serpro69/kotlin-faker

They have a pretty nice table, and already a benchmark it seems. Would be nice to know how Datafaker is performing in this maybe?

**Comment by snuyanzin** [comment]
Yes, I thought about it. The problem here is that there are not so many points by which we could compare since different libs provide different functionalities.... 
I add Kotlin Faker to this page

**Comment by bodiam** [comment]
True, though at least kotlin faker seems to be using the same source yml files as Datafaker.net. I read your article, very interesting. Seems like kotlin faker has the same initialization problem as Java faker. It's just crazy to load all the yml files on startup.

File: docs/documentation/performance.md [hunk 1]
```diff
@@ -1,74 +1,231 @@
...
+Unfortunately there is not so much information what kind of benchmarks have been executed. After some search across the project I was able to find only [this page](https://github.com/serpro69/kotlin-faker/blob/master/docs/src/orchid/resources/pages/faker-comparisons.md)
+So, it looks like the only test which were done is checking `Faker.name().name()` performance. Ok, let's start with the similar test here. Similar, because we are going to use JMH which was not used in their test.
+If applicable we try to execute same tests we did for previous section. So let's start with the original test from Kotlin Faker
+
+Original Test:
+=== "Java"
+
+```java
+    @Benchmark
+    @BenchmarkMode(Mode.Throughput)
```
Comment (bodiam) [inline docs/documentation/performance.md:183]: No results of this test?

Comment (snuyanzin) [reply docs/documentation/performance.md:183]: in the table below.
The only reason I put here 2 tests in one is that they have different clasnames and `getName`/`name` methods
like `io.github.serpro69.kfaker.Faker faker = new io.github.serpro69.kfaker.Faker();` vs `net.datafaker.Faker faker = new net.datafaker.Faker();` and `faker.getName().name();` vs `faker.name().name();`

**Comment by snuyanzin** [comment]
Added a project https://github.com/snuyanzin/datafaker-benchmark and some other metrics.

**Comment by bodiam** [comment]
Would it be an idea to move that repo to datafaker-net/datafaker-benchmark ? I'm happy to make repo if you can't?

**Comment by snuyanzin** [comment]
No objections from my side
However I don't have enough grants to do that

**Comment by bodiam** [comment]
Sorry about that, I'll see if I can fix that. In the meantime, I created a repo for it

**Comment by snuyanzin** [comment]
no problem, thanks for creating it.
Meanwhile I noticed that Kotlin Faker supports some expressions (they call it random string from templates) https://serpro69.github.io/kotlin-faker/wiki/extras/#random-strings-from-templates

Thus, I will try to add comparison for that as well

**Comment by bodiam** [comment]
want me to deploy this documentation?

**Comment by snuyanzin** [comment]
Not yet, still in updating it

**Pull request merged**